web/analytics/views.py

```
@@ -0,0 +1,427 @@
+        has_access = Enrollment.objects.filter(
+            student=student,
+            course__teacher=request.user
+        ).exists()
+        
+        if not has_access:
+            return HttpResponse("Permission denied", status=403)
+    
+    # Route to appropriate export function
+    if export_format == 'csv':
+        return export_csv(request, analytics_type, obj_id)
+    elif export_format == 'pdf':
```

Ruff (0.8.2)

Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)